src/qibolab/_core/instruments/qblox/config.py

```diff
@@ -102,8 +106,17 @@ def module(
     for probe, lo in los:
         if lo is None:
             continue
-        n = mod_channels[probe].ports[0]  # TODO: check it is the correct path
-        getattr(mod, f"out{n}_lo_freq")(cast(OscillatorConfig, configs[lo]).frequency)
+        n = mod_channels[probe].ports[0] - 1
+        if mod.is_qrm_type:
+            attr = f"out{n}_in{n}_lo_freq"
+        else:
+            attr = f"out{n}_lo_freq"
+        getattr(mod, attr)(cast(OscillatorConfig, configs[lo]).frequency)
```

Comment:
Otherwise
```py
AttributeError: 'Module' object and its delegates have no attribute 'out0_lo_freq'. Did you mean: 'out0_in0_lo_freq'?
```
Also same as above, the port numbering seems to start from 0.